src/topology/algebra/ordered.lean

```diff
@@ -1203,6 +1203,41 @@ end
 
 end linear_order
 
+section linear_ordered_ring
+variables [topological_space α] [linear_ordered_ring α] [order_topology α]
+variables {l : filter β} {f g : β → α}
+
+/- TODO The theorems in this section ought to be written in the context of linearly ordered
+(additive) commutative groups rather than linearly ordered rings; however, the former concept does
+not currently exist in mathlib. -/
+
+/-- In a linearly ordered ring with the induced topology, if `f` tends to `C` and `g` tends to
+`at_top` then `f + g` tends to `at_top`. -/
+lemma tendsto_at_top_add_tendsto_left
+  {C : α} (hf : tendsto f l (nhds C)) (hg : tendsto g l at_top) :
+  tendsto (λ x, f x + g x) l at_top :=
```

Comment:
Is there any reason not to use `f + g`?
```suggestion
  tendsto (f + g) l at_top :=
```

Reply:
To use `f + g`, one would have to import `algebra.group.pi`.  In the other file where you pointed this out, there are pre-existing lemmas using `λ x, f x + g x`, so someone made an active decision to postpone that import.  In this file, there are no such lemmas, so the import has not been considered before; I can do it if you like.

Reply:
When I wrote some of the lemmas about algebraic operations on `at_top`, the reason to avoid `f + g` was not a missing `import` (BTW, at this time it was `algebra/pi_instances`, depending on `algebra/module`) but diamonds in algebraic instances for `pi` types.